features/step_definitions/filtering_steps.rb

```diff
@@ -787,6 +792,22 @@
   click_link(link_text)
 end
 
+And(/^I visit the benefits-reform page$/) do
+  visit finder_path('government/policies/benefits-reform')
+end
+
+Then(/^I should see results and pagination$/) do
+  expect(page).to have_text('20 reports')
+  expect(page).to have_css('.finder-results', visible: true)
+  expect(page).to have_css('.gem-c-pagination')
+end
+
+Then(/^I should not see results and pagination$/) do
+  expect(page).not_to have_text('20 reports')
```

Comment:
Hmm, looks like this step should actually be `Then the results and pagination should be removed`, as that is what we're testing.

We're not concerned with the results page not having the text '20 reports', per sé - we're concerned that the page _should_ have the text '20 reports' and then we perform an action that means that text should no longer be there.